examples/yacat/yacat.py

```diff
@@ -70,6 +70,7 @@ async def worker_find_password(ctx: WorkContext, tasks):
 
             # Commands to be run on the provider
             commands = (
+                "rm /golem/work/*.potfile ~/.hashcat/hashcat.potfile || true; "
```

Comment:
I'd suggest `rm -f` instead of `|| true`

Reply:
right, updating.